sycl/test-e2e/Matrix/Legacy/joint_matrix_half.cpp

```diff
@@ -7,6 +7,7 @@
 //===----------------------------------------------------------------------===//
 // REQUIRES: aspect-fp16
 // REQUIRES: matrix
+// REQUIRES: matrix-fp16
```

Comment:
how are you making sure this will run on the GPU as well?
does matrix-fp16 include PVC and DG2?

Reply:
Yep, I see that matrix-xmx8 will imply matrix-fp16, but for PVC we would need to always pass one more param in addition to matrix. Can we use runtime query `device.get_info<info::device::matrix_combinations>();` instead, as it seems complexity of managing those params manually is increasing?

Reply:
These are the only ones we need for 2024.1
matrix-tf32 and matrix-fp16. We should not keep adding these as this is not scalable.

Reply:
What about next releases? ;) Exactly we should stop adding this....